### PR TITLE
Disabled datepicker

### PR DESCRIPTION
**Pull request by jarppe**
Add event listener to eat clicks if disabled? is set.

### CODE REVIEW

File: src/cljs/lomakkeet/reagent/datepicker.cljs [hunk 1]
```diff
@@ -57,15 +67,16 @@
           [:input.form-control
```
Comment (Deraen) [inline src/cljs/lomakkeet/reagent/datepicker.cljs:57]: I wonder if the click handler could be attached to DOM here? Probably Pikaday would overwrite it? Another option would be to wrap input in another element, but that might mess up styles?

File: src/cljs/lomakkeet/reagent/datepicker.cljs [hunk 2]
```diff
@@ -74,26 +85,26 @@
...
+        value (reaction (get-in @form-value ks))]
     (fn [_ {:keys [ks datepicker-i18n min-date max-date date-time? disabled]}]
-      [date {:value @value
-             :on-blur #(impl/blur form ks)
-             :on-clear (fn [e]
-                         ;; Set date to nil
-                         (impl/cb form ks nil))
-             :on-select (fn [date]
-                          (impl/cb form ks date))
+      [date {:value           @value
```
Comment (Deraen) [inline src/cljs/lomakkeet/reagent/datepicker.cljs:88]: I prefer keeping map values (and let forms) unaligned, but not a huge issue. Aligning these causes lots of unnecessary changes when adding/renaming keys.

File: src/cljs/lomakkeet/reagent/datepicker.cljs (hunk 1 above)
Comment (Deraen) [inline src/cljs/lomakkeet/reagent/datepicker.cljs:57]: Ah, the input is already wrapped in `div.input-group` so it should be definitely possible to attach event  handler to that.

**Comment by jarppe** [comment]
Thanks Deraen, using 'pointer-events: none' to disable clicks is way better. TIL.
